File: Java/Program/Shirt.java
public class Shirt extends Clothing {
    
    private String color;
    private String sleeve_type;
    
    public Shirt(int idProduct, String name, String brand, int price, String size, String material, String gender, String color, String sleeve_type) { 
        super(idProduct, name, brand, price, size, material, gender);
        this.color = color;
        this.sleeve_type = sleeve_type;
    }

    public String getColor() {
        return color;
    }
    public void setColor(String color) {
        this.color = color;
    }

    public String getSleeve_type() {
        return sleeve_type;
    }
    public void setSleeve_type(String sleeve_type) {
        this.sleeve_type = sleeve_type;
    }
    
    public void showInfo() { 
        super.showInfo();
        System.out.println("Color: " + color);
        System.out.println("Sleeve type: " + sleeve_type);
    }
}

File: Python/Program/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):
    def __init__(self, id_product, name, brand, price, size, material, gender, color, sleeve_type):
        super().__init__(id_product, name, brand, price, size, material, gender)
        self.color = color
        self.sleeve_type = sleeve_type

    def get_color(self):
        return self.color

    def set_color(self, color):
        self.color = color

    def get_sleeve_type(self):
        return self.sleeve_type

    def set_sleeve_type(self, sleeve_type):
        self.sleeve_type = sleeve_type
